ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/backup/AndroidBackup.kt

```diff
@@ -0,0 +1,73 @@
+package co.electriccoin.zcash.ui.screen.backup
+
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.content.Context
+import androidx.activity.ComponentActivity
+import androidx.activity.viewModels
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import cash.z.ecc.sdk.model.PersistableWallet
+import co.electriccoin.zcash.ui.MainActivity
+import co.electriccoin.zcash.ui.R
+import co.electriccoin.zcash.ui.screen.backup.ext.Saver
+import co.electriccoin.zcash.ui.screen.backup.state.BackupState
+import co.electriccoin.zcash.ui.screen.backup.state.TestChoices
+import co.electriccoin.zcash.ui.screen.backup.view.BackupWallet
+import co.electriccoin.zcash.ui.screen.backup.viewmodel.BackupViewModel
+
+@Composable
+internal fun MainActivity.WrapBackup(
+    persistableWallet: PersistableWallet,
+    onBackupComplete: () -> Unit
+) {
+    WrapBackup(this, persistableWallet, onBackupComplete)
+}
+
+// This layer of indirection allows for activity re-creation tests
+@Composable
```

Comment:
No changes needed here.  Informational only.  

This method is basically only visible for integration testing purposes.